src/couch_replicator_scheduler_job.erl

```diff
@@ -450,11 +450,8 @@ format_status(_Opt, [_PDict, State]) ->
     [{data, [{"State", state_strip_creds(State)}]}].
 
 
-report_job_error(_Rep, _Error) ->
-    % TODO: handle errors back to job scheduler to let it
-    % decide if needed to backoff and retry and what the back-off schedule
-    % should be
-    ok.
+report_job_error(Rep, Error) ->
+    exit({error, Rep, Error}).
```

Comment:
Tested locally. Killed a replication by hand noticed that keeping `#rep{}` record in each crash event could be a problem. I leaks credential to logs and is a rather large chunk of data (compared to other things there).